instrumentation/faraday/lib/opentelemetry/instrumentation.rb

```diff
@@ -5,12 +5,12 @@
 # SPDX-License-Identifier: Apache-2.0
 
 module OpenTelemetry
-  # "Instrumentation adapters" are specified by
+  # "Instrumentation instrumentations" are specified by
```

Comment:
copypasta noted above.

Reply:
okey i've updated this and it's other usage to `"Instrumentation" are specified by`...open to anything else if that's preferred